RecordCollection/Controllers/AlbumsController.cs

```diff
@@ -15,8 +19,39 @@ public AlbumsController(RecordCollectionContext context)
         public IActionResult Index()
         {
             var albums = _context.Albums.ToList();
-            
             return View(albums);
         }
+
+        [Route("Albums/{id:int}")]
+        public async Task<IActionResult> Show(int? id)
+        {
+            if (id == null)
+            {
+                return NotFound();
+            }
+
+            var retrieved = await _context.Albums.FindAsync(id);
+
```

Comment:
It would be good to add null checking for the retrieved album here (~line 34), in case the user inputs a valid id that doesn't match any albums in the database